faq section change

chait04 opened the pull request:
## Issue that this pull request solves

 Closes: #216

## Proposed changes
 * Make the image and card containing FAQs one below another rather than side by side
 * Give hover effect to the Faq card (#016795)

### Brief description of what is fixed or changed
* added hover effect on faq card
* changed faq section in column

## Types of changes

_Put an `x` in the boxes that apply_

- [x ] Bugfix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

mohit355 commented:
Include some SnacpShot

mohit355 commented on frontend/src/components/faq/faq.css:
```diff
@@ -45,6 +46,12 @@
   ) !important;
```
Using important is not the best practice. Try to avoid it

mohit355 replied:
> ** Now the final version look like this **
> ` After adding padding as @Kajol-Kumari said and solving the border issue as @mohit355 said `
> ![Screenshot (189)](https://user-images.githubusercontent.com/59140343/103992017-852c3a80-51b9-11eb-91b6-7ef1e659712b.png)
> 
> ` In case u are wondering why the first and last card dont have curve radius on its top and bottom respectively .Its because of material ui ... anyways it looks good`
> 
> ![Screenshot (190)](https://user-images.githubusercontent.com/59140343/103992021-865d6780-51b9-11eb-971b-995b23b70564.png)

@Kajol-Kumari  is this type of border-radius ( for first and last div) look good to you?  Not looks good to me.

chait04 replied:
I can add radius to them also( first div & last div )   ..Should I add?..

mohit355 reviewed (changes requested):
Fix the hover effect that breaks at corners during removing the cursor from the accord.

Kajol-Kumari commented on frontend/src/components/faq/faq.css:
```diff
@@ -24,19 +25,19 @@
   height: auto;
   border-radius: 8px;
-  padding: 10px;
+  padding: 15px 16px;
```
let's keep it `padding 15px 40px`

Kajol-Kumari reviewed (changes requested):
Hey @chait04 apart from what @mohit355 suggested please do the following
1. Add the title and image side-by-side(image on the left, title on the right, it will become one below other for smaller screens)
2. After the accordion get's opened, make the hover effect to the whole card,instead of just title

Rest looks good to me :)

chait04 commented:
I changed the css and added effects using material ui...
https://user-images.githubusercontent.com/59140343/103990783-c4598c00-51b7-11eb-9823-0f930399d4db.mp4

chait04 commented:
@mohit355 now the hover effect is working properly
![Screenshot (185)](https://user-images.githubusercontent.com/59140343/103991109-4053d400-51b8-11eb-9076-098171e0f6fd.png)
I added it using material ui styling
![Screenshot (186)](https://user-images.githubusercontent.com/59140343/103991111-41850100-51b8-11eb-8ef0-3c0f4544b133.png)

chait04 commented:
<strong> Now the final version look like this </strong>
 <code> After adding padding as @Kajol-Kumari said and solving the border issue as @mohit355 said </code>
![Screenshot (189)](https://user-images.githubusercontent.com/59140343/103992017-852c3a80-51b9-11eb-91b6-7ef1e659712b.png)

<code> In case u are wondering why the first and last card dont have curve radius on its top and bottom respectively .Its because of material ui ... anyways it looks good</code>

![Screenshot (190)](https://user-images.githubusercontent.com/59140343/103992021-865d6780-51b9-11eb-971b-995b23b70564.png)

Kajol-Kumari reviewed (changes requested):
please remove [this](https://github.com/HITK-TECH-Community/Community-Website/blob/cb8134f3ed4a812a4e686db724e8fd8fb99dc041/frontend/src/components/faq/faq.css#L1) style and also [this](https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/src/components/faq/faq.css#L8) one

Also let's keep the title and image side by side like this
![Screenshot from 2021-01-08 18-24-03](https://user-images.githubusercontent.com/44888949/104017854-de5a9500-51de-11eb-9647-fcad01357420.png)

Kajol-Kumari commented:
@chait04 please implement these minor changes and it will be good to go :)

chait04 commented:
<strong> desktop view </strong>
![Screenshot (199)](https://user-images.githubusercontent.com/59140343/104024527-42825680-51e9-11eb-821d-a039aad7048d.png)
<strong>Mobile view </strong>
![Screenshot (198)](https://user-images.githubusercontent.com/59140343/104024519-41512980-51e9-11eb-97a8-5ddb869cead2.png)

Kajol-Kumari reviewed (approved):
Thanks for incorporating the suggested changes @chait04 
LGTM :+1:

Kajol-Kumari commented:
@chait04 Please make sure that all your merged PRs have appropriate labels added to it. Incase we miss any, let us know.
Happy Learning :)
